include/openthread/mdns.h

```diff
@@ -335,7 +353,7 @@ otError otMdnsUnregisterService(otInstance *aInstance, const otMdnsService *aSer
  *
  * The fields in @p aKey follow these rules:
  *
- * - If the key is associated with a host entry, `mName` specifies the host name and `mServcieType` MUST be NULL.
+ * - If the key is associated with a host entry, `mName` specifies the host name and `mServiceType` MUST be NULL.
  * - If the key is associated with a service entry, `mName` specifies the service instance label (always treated as
  *   a single label) and `mServiceType` specifies the service type (e.g., "_tst._udp"). In this case the DNS name for
  *   key record is `<mName>.<mServiceTye>`.
```

Comment:
If you're fixing typos in this PR, mServiceTye -> mServiceType

Here and just below

Reply:
I fixed them all in new push.
There were more of same typo in the comments in `cpp` file and variable names (searched and replaced them all in the code).